Packs/Illumio/.pack-ignore

```diff
@@ -0,0 +1,2 @@
+[file:IllumioCore_image.png]
+ignore=IM111
```

Comment:
please try to fix it, it shouldn’t be something hard to fix. all you need is probably just reaching one of your designers. it should be very easy for them to fix it.